### PR TITLE
Adding rubucop instruction.

### PR DESCRIPTION
**Pull request by banurekha**
Adding rubucop instruction to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,3 +48,18 @@ The following will run the `spec/curate/functional/func_curate_spec.rb` against
 cd /path/to/QA_tests
 ENVIRONMENT=prod rspec spec/curate/functional/func_curate_spec.rb
 ```
+
+## Running Rubocop
+
+Rubocop is install and configured to report any offenses in codeing. It is recommended to run rubocop script before sending in pull request. Here is basic usage
+
+```
+cd git/QA_tests
```
Comment (h-parekh) [inline README.md]: Replace with 'cd /path/to/QA_tests'

File: README.md [hunk 2]
```diff
@@ -48,3 +48,18 @@ The following will run the `spec/curate/functional/func_curate_spec.rb` against
 cd /path/to/QA_tests
 ENVIRONMENT=prod rspec spec/curate/functional/func_curate_spec.rb
 ```
+
+## Running Rubocop
+
+Rubocop is install and configured to report any offenses in codeing. It is recommended to run rubocop script before sending in pull request. Here is basic usage
```
Comment (h-parekh) [inline README.md]: Update as: It is recommended to run rubocop script **on local machine** before sending in pull request

File: README.md [hunk 3]
```diff
@@ -48,3 +48,19 @@ The following will run the `spec/curate/functional/func_curate_spec.rb` against
 cd /path/to/QA_tests
 ENVIRONMENT=prod rspec spec/curate/functional/func_curate_spec.rb
 ```
+
+## Running Rubocop
+
+Rubocop is install and configured to report any offenses in code. It is recommended to run rubocop on local machine before sending in pull request. Here is basic usage
+
+```
```
Comment (jeremyf) [inline README.md]: Add the console context

    ```console

This helps differentiate the type of code that is being demonstrated

File: README.md [hunk 4]
```diff
@@ -48,3 +48,19 @@ The following will run the `spec/curate/functional/func_curate_spec.rb` against
...
+
+```console
+cd path/to/QA_tests
+bundle exec rubocop
+```
+
+It is possible to auto-correct certain offenses. it is experimental so use it with caution.
+
+```console
+cd path/to/QA_tests
```
Comment (jeremyf) [inline README.md]: Please include the leading `/`

```console
cd /path/to/QA_tests
```

File: README.md [hunk 5]
```diff
@@ -48,3 +48,19 @@ The following will run the `spec/curate/functional/func_curate_spec.rb` against
 cd /path/to/QA_tests
 ENVIRONMENT=prod rspec spec/curate/functional/func_curate_spec.rb
 ```
+
+## Running Rubocop
+
+Rubocop is install and configured to report any offenses in code. It is recommended to run rubocop on local machine before sending in pull request. Here is basic usage
+
+```console
```
Comment (jeremyf) [inline README.md]: Please include the leading `/` as it is a somewhat unspoken expectation when talking about the path to a file; (The leading slash means start at the root of the machine. Without the leading slash it means starting wherever you currently are)

```console
cd /path/to/QA_tests
```

**Pull request merged**